static/app/views/replays/details.tsx

```diff
@@ -22,12 +24,13 @@ type Props = RouteComponentProps<
   {orgId: string; replaySlug: string},
   {},
   any,
-  {t: number}
+  {event_t: string; t: number}
 >;
 
 function ReplayDetails({
   location: {
     query: {
+      event_t: eventTimestamp, // Timestamp of the event or activity that was selected
```

Comment:
after this PR lands should we prefer `event_t` over `t`? it seems easier to create inbound links with this new queryparam

Reply:
Well `event_t` takes a timestamp and calculates the initial time offset with it, instead `t` takes in the initial time offset in seconds already calculated

Reply:
that's right. so in order to use `t` some code not on this page would have to start with a timestamp, and would need to calclulate the offset independantly.

should we instead always pass `event_t` and then no other pages will need to implement that calculation?

Reply:
Oh! Yeah that makes a lot of sense, I agree - we should do a follow up PR for this for sure